MHIdentityKit/OAuth2/DefaultAccessTokenRefresher.swift

```diff
@@ -63,6 +63,23 @@ open class DefaultAccessTokenRefresher: AccessTokenRefresher {
             }
         }
     }
+    
+    @available(iOS 13, tvOS 13.0.0, macOS 10.15, *)
+    open func refreshAsync(using requestModel: AccessTokenRefreshRequest) async throws -> AccessTokenResponse? {
```

Comment:
The result should not be optional. It should throw an error (eg. `MHIdentityKitError.Reason.invalidAccessTokenResponse`) if there is no `AccessTokenResponse`.

This should be applied to all similar cases.